app/grandchallenge/components/models.py

```diff
@@ -554,6 +554,13 @@ def file_extension(self):
         except KeyError as e:
             raise RuntimeError(f"Unknown kind {self.kind}") from e
 
+    @property
+    def custom_upload_processing_queue(self):
+        try:
+            return INTERFACE_KIND_TO_CUSTOM_QUEUE[self.kind]
+        except KeyError:
+            return None  # No custom queue: use default
```

Comment:
Probably doesn't even need a method!

```suggestion
        return INTERFACE_KIND_TO_CUSTOM_QUEUE.get(self.kind)
```